docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -1020,18 +1001,11 @@ customresourcedefinition.apiextensions.k8s.io "csistoragecapacities.storage.k8s.
 
 ## Removing a Workload Management Workspace
```

Comment:
Keep the wording of this section title parallel with the wording of the title of the section about creating a WMW/WDS.